tembo-cli/tests/tomls/set/tembo.toml

```diff
@@ -0,0 +1,9 @@
+[defaults]
```

Comment:
Can you use a different name here? This is for the same reason mentioned above. Also, can you move this under the examples directory?